build.gradle

```diff
@@ -695,7 +695,13 @@ project(':streams') {
       exclude module: 'jline'
       exclude module: 'netty'
     }
-    compile libs.jacksonDatabind // this dependency should be removed after KIP-4
+
+    // zkclient has a hard log4j dependency, we set it here as well to avoid copying an older
+    // version to the binary package; the following three dependencies should be removed after KIP-4
+    // (along with the zkclient dependency)
+    compile libs.log4j
+    compile libs.slf4jlog4j
+    compile libs.jacksonDatabind
```

Comment:
It is updated in the added comment `the following three dependencies should be removed after KIP-4`.